src/browser/extension.ts

```diff
@@ -11,12 +11,13 @@ import { Commands } from '../commands';
 import { DebugTracker } from '../debug-tracker';
 import { SvdRegistry } from '../svd-registry';
 import { SvdResolver } from '../svd-resolver';
+export * from '../export';
```

Comment:
This is a little ugly. A different approach could be to not export here and instead guide users to import the types directly. e.g. Have the interfaces extracted to a file (e.g. `api.ts`) and update the docs to use:

```typescript
import type * as api from from 'vscode-svd-viewer/api';
```

To be used as `api.PeripheralOptions`...

Reply:
I made the change with a little bit different approach. I moved the type definitions to a separate ile called `api.ts` but I avoid to add the `api.ts` to the root folder so I kept exporting here but encapsulated it as `api`. 

Also we are unable to use `import type` at the client because `AccessType` is an enum and this is preventing the use of `import enum`

Final client usage will be like below:

```js
import { api } from 'svd-viewer';
```

How this sounds to you?